src/game.zig

```diff
@@ -509,6 +509,12 @@ pub const collision = struct {
 
 						entity.touchingClimbable = entity.touchingClimbable or isSideBlock or isBottomBlock;
 						entity.climbing = isSideBlock and (!isBottomBlock or isFullBlock);
+
+						const normal = if (model.internalQuads.len > 0) model.internalQuads[0].quadInfo().normal else break :missingNormal;
```

Comment:
Wait, so you cannot make full blocks climbable? That doesn't make sense.
Also does this work on other blocks, like e.g. if you were to make a climbable pole?